configure.ac

```diff
@@ -507,7 +507,7 @@ AC_MSG_RESULT([
 	testdir:		${testdir}
 	WITH_ARCH:              ${WITH_ARCH}
 
-	cc:			${CC}
+	cc:                     ${CC}
```

Comment:
This is an unnecessary white space change and the only change in this file. There is no need for it.

Reply:
When we had first created the PR, checkpatch was throwing a false positive because it would confuse it with the email CC field. Petri and Matias had agreed to ignore this warning but I think we had forgotten to get rid of this change. I will remove it now. Thanks!